### PR TITLE
Fixes rpgloot.dm by removing + and -

### LINKED ISSUE #4708: Virus storage doesn't vend items with special characters #, %, & in their name
#, % and & don't vend

 (, ? and ´ do vend

I put in my virus but can't get it out, :-(

### LINKED ISSUE #11779: Magic event breaks Smart Virus Storage
The magic event that adds names to shit breaks the items inside Smart Virus Storage (probably applies to all fridges). The items are listed but when clicked to withdrawn, they are gone.

### PR DESCRIPTION
**Pull request by Jalleo**
and replaces it with a plus and a space

This dosent solve the issue of when the item is already in a fridge but it allows it to be stored.

Should fixes #4708
fixes #11779

### CODE REVIEW

**Comment by Incoming5643** [comment]
Having machines that are reliant on an item's name being formated in a certain way is the problem here, not this.

**Comment by Jalleo** [comment]
I was working on fixing that but it took quite a while to work out what was stopping the vending because literally a single locate was needed

File: data/mode.txt [hunk 1]
```diff
@@ -1 +1 @@
-extended
\ No newline at end of file
+secret
```
Comment (Cheridan) [inline data/mode.txt]: secretssss

Comment (Jalleo) [inline data/mode.txt]: whoops accidental addition due to other stuff removing that now

**Comment by ike709** [comment]
Pretty sure you need "Fixes #Issuenumber" before both of them in the OP or it won't autoclose them both.

**Comment by Jalleo** [comment]
Not certain if it completely fixes it yet need to do thorough testing

**Comment by Iamgoofball** [comment]
Remove the removals of + and - if you fixed the actual bug nerd

File: data/mode.txt [hunk 2]
```diff
@@ -1 +0,0 @@
-extended
```
Comment (MrStonedOne) [inline data/mode.txt]: ???

Comment (Jalleo) [inline data/mode.txt]: Ill fix that later today

**Comment by Jalleo** [comment]
Still trying to fix it up @jjrcop helped me earlier but underlying issue not fixed yet should be soon hopefully.

File: data/mode.txt (hunk 2 above)
Comment (Jalleo) [inline data/mode.txt]: I have fixed it github is being a derp and not realised I have

**Comment by Jalleo** [comment]
@Iamgoofball nope it turns out just changing the names was breaking smart fridge code I am keeping those changes just to prevent another issue not related to the smart fridge code.

**Comment by Jalleo** [comment]
Only issue left is smartfridges no longer stack stuff not sure exactly  how to fix yet but will do.
Thank you to @MrPerson for helping me over the final issue with it not vending properly.

**Comment by JJRcop** [comment]
I misunderstood the code!! I am a horrible person, my "fix" was terrible! Why do I even think?!

**Comment by Jalleo** [comment]
Right after having a convosation with @MrStonedOne and others its best to leave this PR as it is and at a later date (Next week hopefully) I come back with another PR to refactor the whole smartfridge code by breaking it up and making it more friendly for the different areas that use it.

**Pull request merged**